bootstrap/sql/postgresql/create_tables.sql

```diff
@@ -0,0 +1,344 @@
+-- THE NAMES OF THE TABLE COLUMNS MUST MATCH THE NAMES OF THE CORRESPONDING CLASS MODEL FIELDS;
+
+CREATE TABLE IF NOT EXISTS dashboard (
```

Comment:
Old names were used for compatibility with Pojos and for testing. After https://github.com/hortonworks/streamline/pull/512 is merged, I will rename these.